packages/@react-spectrum/progress/stories/ProgressBar.stories.tsx

```diff
@@ -37,11 +29,22 @@ const grayedBoxStyle: CSSProperties = {
 };
 
 storiesOf('Progress/ProgressBar', module)
-  .addParameters({providerSwitcher: {status: 'positive'}})
-  .addDecorator(withKnobs)
+  .addParameters({
+    providerSwitcher: {status: 'positive'},
+    args: {value: 32},
+    argTypes: {
+      value: {
+        control: {
+          type: 'range',
+          min: 0,
+          max: 100
+        }
+      }
+    }
+  })
   .add(
     'Default',
-    () => render()
+    (args) => render(args)
   )
   .add(
     'value: 50',
```

Comment:
Would we eventually want this story to work with controls?

Reply:
Seems weird to have a hardcoded value of 50 in the story, but allow it to be edited in controls also?

Reply:
I just edited my comment to suggest removing the hardcoded 50 and 100 stories without realizing you'd replied.